ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-release-3.0.yaml

```diff
@@ -87,16 +87,16 @@ tests:
     export GOCACHE="/tmp/cache"
     export GOMODCACHE="/tmp/cache"
     export GITHUB_TOKEN_PATH=/creds-github/token
-    export MERGE_BRANCH=release-0.1
+    export MERGE_BRANCH=release-3.0
     export VERSIONS_YAML_FILE="ossm/versions.yaml"
     ./tools/automator-main.sh \
     --org=openshift-service-mesh \
     --repo=sail-operator \
-    --branch=release-3.0-tp1 \
+    --branch=release-3.0 \
     --token-path=/creds-github/token \
     '--title=Automator: merge upstream changes to $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH' \
     "--labels=auto-merge,tide/merge-method-merge" \
-    --modifier=merge_upstream_release-0.1 \
+    --modifier=merge_upstream_release-3.0 \
     --cmd=./ossm/merge_upstream.sh
```

Comment:
Missing this folder and scripts in :
https://github.com/openshift-service-mesh/sail-operator/tree/release-3.0

PR fix: https://github.com/openshift-service-mesh/sail-operator/pull/208

Reply:
It's also fixed in https://github.com/openshift-service-mesh/sail-operator/pull/207 ;)